lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -77,10 +77,27 @@ def on_block(node)
         private
 
         def example_with_aggregated_failures?(node)
-          example = node.send_node
+          aggregate_failures = aggregated_failures?(node)
 
-          with_aggregated_failures?(example) &&
-            !disabled_aggregated_failures?(example)
+          aggregate_failures ||
```

Comment:
this is a weird pattern (making distinctions between false and nil). Perhaps aggregated_failures? should perform the search up in the three, in case it's not set in the current node, and just return true/false

Reply:
Fair enough. Reworked the search to a recursive method.